### ISSUE TITLE
WIN 10 x64/ARM64 versions

### ISSUE THREAD

**Issue by NHarishGit** [impleotv/gstreamer-klv-plugins-release#1]
Do you support Windows 10/11 x64 and ARM64 builds for klvdecode and klvencode DLLs ?
Would like to try them

**Comment by impleotv** [comment]
We're having troubles compiling GStreamer and plug-ins for Windows. If you know how to do that and could help us, we would be happy to  provide the dlls. 

Alex

**Comment by NHarishGit** [comment]
YEs, I can handle that, have been successfully compiling and executing Gstreamer for WINDOWS platforms. Let me know how can we colloborate.

**Comment by impleotv** [comment]
Great. BTW, you will also need to fix the bug in the GStreamer demux, otherwise it won't work with SYNC KLV. I'll send you the patch tomorrow.

**Comment by NHarishGit** [comment]
Let me know, if we have to sign a NDA, impleoTV IP source code under
klvdecode and klvencode dlls will be protected.

Hopefully, we can arrange for my access soon and I can integrate and build
them for Windows targets and test it with async data in the meantime, while
you can send me the "tsdemux" fix for absorbing sync klv data

Regards
Nitin

**Comment by impleotv** [comment]
Here is the patch for demux:
[sync-klv-tsdemux -1.20.5.patch](https://github.com/impleotv/gstreamer-klv-plugins-release/files/10576590/sync-klv-tsdemux.-1.20.5.patch)
It is not ideal but works for us. Some things should be added, but we don't have enough expertise with GStreamer (and GStreamer guys are not that collaborative).
Please send me a mail (to alexc@impleotv.com), so I could provide you with further instructions.

Thanks,

Alex
